src/Sentry.Bindings.Android/Sentry.Bindings.Android.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>net7.0-android</TargetFramework>
     <!-- BG8605 and BG8606 happen because there's a missing androidx.lifecycle dependency, but we don't need it here.  (The native Android Sentry SDK will use it if it exists.) -->
     <NoWarn>$(NoWarn);BG8605;BG8606</NoWarn>
-    <SentryAndroidSdkVersion>7.15.0</SentryAndroidSdkVersion>
+    <SentryAndroidSdkVersion>7.15.0-alpha.1</SentryAndroidSdkVersion>
```

Comment:
We changed the SdkVersion to `7.15.0-alpha.1` just for the alpha-release and to make it easy for CI. We can remove it just before merging to `main`

[more](https://github.com/getsentry/sentry-dotnet/pull/3694#issuecomment-2430134945)

Reply:
OK, is the plan to wait for this to be shipped in the main branches of the native and android libraries first then? Have we confirmed this resolves the original issue that it was meant to address?

Reply:
OK cool. So maybe we don't merge this into main just yet... we can make an alpha release from this branch "so folks can try that out".

Thanks heaps @bricefriha - nice work!

Reply:
with this we're markign only this package as alpha. How do we release from the branch? If I set the whole repo's version `-alpha`, this package will be `-alpha0.1-alpha0.1`.

Reply:
@bruno-garcia I thought this indicated we depend on an alpha of the Sentry Android SDK (it doesn't mark any of the packages in the sentry-dotnet repo as alpha). Is that not the case?

Reply:
lets make an alpha of this branch alone (the whole SDK can be alpha).
otherwise we need to merge into `main` and if we make previews from there, we'll have teh issue I mentioend